src/telemetry/metrics.yaml

```diff
@@ -12,20 +12,497 @@ page:
   view:
     type: event
     description: |
-      An event triggered when the page is loaded.
+      A page view event e.g trackable by the URL.
     bugs:
-      - https://mozilla-hub.atlassian.net/browse/MNTOR-2022
+      - https://mozilla-hub.atlassian.net/browse/MNTOR-2399
     data_reviews:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1848664#c6
+      - TODO
     data_sensitivity:
       - interaction
     notification_emails:
       - rhelmer@mozilla.com
     expires: never
     extra_keys:
-      path:
-        description: The path of the page.
+      url:
+        description: The full URL of the page.
         type: string
       user_id:
-        description: User ID for Nimbus. Derived from subscriber ID or randomly-generated if not logged in.
+        description: Mozilla accounts user ID. Randomly-generated if not logged in.
         type: string
+      dashboard_tab:
+        description: Either [action_needed, fixed] depending on where the user is in their dashboard.
+        type: string
+      legacy_user:
+        description: This date is used to direct users who signed up after data broker scanning was released to the welcome flow.
+        type: boolean
+      breach_count:
+        description: Users who run the new free scan will be notified of breaches found during the scan.
+        type: quantity
+      legacy_breach_count:
+        description: Legacy users of Monitor may already have breaches reported on their dashboard, without having run the new free scan.
+        type: quantity
+      broker_count:
+        description: Users who run the new free scan will be notified of breaches found during the scan.
+        type: quantity
+      session_id:
+        description: A flow ID (or similar) that allows us to track “sessions” of the user experience within the product.
+        type: string
+      utm_campaign:
+        description: Attribution campaign.
+        type: string
+      utm_content:
+        description: Attribution content.
+        type: string
+      utm_medium:
+        description: Attribution medium.
+        type: string
+      utm_source:
```

Comment:
Should this have a definition for `utm_term` too?
If yes, same comment for:
- L398
- L452
- L506

Reply:
Ugh yes thanks - I expect this would've thrown at runtime (since the instrumentation tries to use it) but thanks for catching it now. I'll look into adding unit tests, I think it would be worthwhile.

Reply:
Also I removed `utm_` from the other events - I think it's only really relevant for page view. I'll check with data science folks on that though.